admin-dev/themes/new-theme/js/pages/order/create/product-manager.js

```diff
@@ -182,7 +182,7 @@ export default class ProductManager {
    * @private
    */
   _search(searchPhrase) {
-    if (searchPhrase.length < 3) {
+    if (searchPhrase.length < 2) {
```

Comment:
Should be define in a const, or with a comment to explain why we set this value (linked to a spec file maybe?)

Reply:
I guess you're right about the spec
@MatShir what do you think? This defines how many characters we need to type before searching for products
In the customer search this was reduced to 1 character previously (nice when you need to look or "jo" 😅), did you write all this down somewhere in the spec?

I hesitated between 1 or 2, but I thought 1 would match too many products. What limit do you think we should use here?

Reply:
Maybe we should look for a debounce instead of min characters :thinking:

Reply:
There's already a debounce of 300ms managed in `CreateOrderPage::_initProductSearch`
And in this _search function the current request (if present) is aborted

Reply:
ping @MatShir